udata/linkchecker/tasks.py

```diff
@@ -53,5 +53,8 @@ def check_resources(self, number):
         resource_obj = get_by(dataset_obj.resources, 'id', rid)
         log.info('Checking resource %s (%s/%s)',
                  resource_id, idx + 1, nb_resources)
-        check_resource(resource_obj)
+        if resource_obj.is_need_check():
+            check_resource(resource_obj)
+        else:
+            log.info("\t-> Skipping this resource, cache is fresh enough.")
```

Comment:
Be careful with ad-hoc logging formating, it won't produce the same result everywhere.

Reply:
👍